elixir/exercicios/accumulate/README.md

```diff
@@ -0,0 +1,68 @@
+# Accumulate
```

Comment:
If you have the time, translate this for our mother tongue to make it easier for our ninjas.